jenkins/job-configs/kubernetes-jenkins/kubernetes-build.yaml

```diff
@@ -3,102 +3,124 @@
     description: 'Grab the latest from GitHub, build. Test owner: Build Cop.'
     node: 'build'
     properties:
-        - build-discarder:
-            num-to-keep: 200
+    - build-discarder:
+        num-to-keep: 200
     builders:
-        - activate-gce-service-account
-        - shell: 'JENKINS_BUILD_STARTED=true bash <(curl -fsS --retry 3 "https://raw.githubusercontent.com/kubernetes/kubernetes/master/hack/jenkins/upload-to-gcs.sh")'
-        - shell: |
-            {job-env}
-            timeout -k {kill-timeout}m {timeout}m ./hack/jenkins/build.sh && rc=$? || rc=$?
-            {report-rc}
+    - activate-gce-service-account
+    - shell: 'JENKINS_BUILD_STARTED=true bash <(curl -fsS --retry 3 "https://raw.githubusercontent.com/kubernetes/kubernetes/master/hack/jenkins/upload-to-gcs.sh")'
+    - shell: |
+        {job-env}
+        timeout -k {kill-timeout}m {timeout}m {script} && rc=$? || rc=$?
+        {report-rc}
     publishers:
-        - gcs-uploader
-        - workspace-cleanup:
-            dirmatch: true
-            exclude: ['.git/']
-            external-deletion-command: 'sudo rm -rf %s'
```

Comment:
I am not sure why we have been deleting the workspace at both the beginning and end of the job, but we can add it back if there are problems.